### ISSUE TITLE
Why i get "loss is nan, but AP/AR is correct"

### ISSUE THREAD

**Issue by miaoshuyu** [open-mmlab/mmdetection#5621]
![image](https://user-images.githubusercontent.com/26992220/125782074-e5177319-aa0b-4372-b309-83eade764951.png)
![image](https://user-images.githubusercontent.com/26992220/125782098-abf654ef-b205-4f69-b1d1-a19a56f8c572.png)

when i train my model, i obtained that the loss is nan, meanwhile AP/AR is right. Who can tell me why?

**Comment by noobying** [comment]
Can you show us the Config file ? 😊

**Comment by miaoshuyu** [comment]
@noobying We train the model on faster_rcnn_r50_fpn.py, what we change is
![image](https://user-images.githubusercontent.com/26992220/125796340-60896998-adaf-4b6a-b38f-6495602d6f38.png)
![image](https://user-images.githubusercontent.com/26992220/125796561-6bbf60e4-ba67-4986-a9a4-717b49456c2b.png)

**Comment by jshilong** [comment]
Seems your learning rate is zero

**Comment by miaoshuyu** [comment]
@noobying @jshilong I try other lr， also meet this issue。
![image](https://user-images.githubusercontent.com/26992220/125894059-ff8f98dd-34a4-48f9-80c2-10419f347ff0.png)

**Comment by BIGWangYuDong** [comment]
Maybe your dataset is too small and this will make the loss to nan.
